x-pack/docs/en/ilm/set-up-lifecycle-policy.asciidoc

```diff
@@ -1,6 +1,110 @@
 [[set-up-lifecycle-policy]]
 == Set up {ilm} policy
 
-Focus on how this is done in the general sense, not the fine points about
-rollover just yet. Specifically that `index.lifecycle.name` can be used here
-too, not just the create index API explicitly.
+In order for an index to use an {ilm} policy to manage its lifecycle we must
+first define a lifecycle policy for it to use. The following request creates
+a policy called `my_policy` in Elasticsearch which we can later use to manage
+our indexes.
+
+[source,js]
+------------------------
+PUT _ilm/my_policy
+{
+  "policy": {
+    "phases": {
+      "hot": {
+        "actions": {
+          "rollover": {
+            "max_size": "25GB" <1>
+          }
+        }
+      },
+      "delete": {
+        "after": "30d",
+        "actions": {
+          "delete": {} <2>
+        }
+      }
+    }
+  }
+}
+------------------------ 
+// CONSOLE
+<1> Rollover the index when it reaches 25GB in size
+<2> Delete the index when its 30 days old
+
+{ilm} will manage an index using the policy defined in the 
+`index.lifecycle.name` index setting. If this setting does not exist in the
+settings for a particular index {ilm} will not manage that index.
+
+To set the policy for an index there are two options:
+1. Apply the policy to an index template and bootstrap creating the first index
+2. Apply the policy to a new index in a create index request
```

Comment:
Should we call out and/or link this to the important note below about rollovers ?